pennylane/qchem/molecule.py

```diff
@@ -138,7 +145,6 @@ def atomic_orbital(self, index):
         coeff = self.basis_set[index].coeff
         r = self.basis_set[index].r
 
-        coeff = coeff * primitive_norm(l, alpha)
         coeff = coeff * contracted_norm(l, alpha, coeff)
```

Comment:
What does this change do?

Reply:
The primitive Gaussians are now normalised in the `Molecule` class and `coeff = coeff * primitive_norm(l, alpha)` is not needed anymore. However,  `contracted_norm` still needs to be applied although it does not really change anything and `contracted_norm(l, alpha, coeff)` will be return one. It is just safer to apply `contracted_norm(l, alpha, coeff)`.